nixos/modules/security/wrappers/wrapper.c

```diff
@@ -38,6 +35,12 @@ static char *wrapper_debug = "WRAPPER_DEBUG";
 #define LE32_TO_H(x) (x)
 #endif
 
+static noreturn void assert_failure(const char *assertion) {
```

Comment:
May be a bit nit-picky, but would it be possible to split out this new `assert_failure` stuff from the "require argc > 0" stuff into separate commits? When skimming the diff, it wasn't immediately obvious to me where this new requirement was encoded... A separate commit that just adds the `ASSERT(argc >= 1);` line would be an improvement, IMHO.

Reply:
That's fair. Fixed.